src/cmd-kola

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+dn=$(dirname $0)
+. ${dn}/cmdlib.sh
+
+latest_qcow=$(get_latest_qemu)
+if [ -z "${latest_qcow}" ]; then
+    fatal "No latest build"
+fi
+
+# XXX: teach to kola
+distro=
+bn=$(basename "${latest_qcow}")
+if [[ ${bn} = fedora-coreos-* ]]; then
+    distro="-b fcos"
+elif [[ ${bn} = rhcos-* ]]; then
+    distro="-b rhcos --ignition-version v2"
+else
+    echo "WARNING: Failed to detect distro, use -b to specify"
+fi
+
+# let's print out the actual exec() call we do
+set -x
+
+# shellcheck disable=SC2086
+exec sudo kola $distro --output-dir tmp/kola --qemu-image "${latest_qcow}" "$@"
```

Comment:
`kola` won't care in that case. It doesn't require unpriv, it just doesn't have any of the bits that required privileged access to setup.

Reply:
Yeah, good point. I dropped the `sudo` since that's the whole point of `qemu-unpriv`. One can always do `sudo cosa kola ...` if they want to run privileged.